pygameMenu/menu.py

```diff
@@ -251,7 +250,6 @@ def __init__(self,
         self._frame = 0
         self._index = 0  # Selected index
         self._onclose = onclose  # Function that calls after closing menu
-        self._size = 0  # Menu total elements

```

Comment:
What happened with this? replaced by len(elements)?

Reply:
the `self._size` counter was used to store the number of elements in the list `self._option` which is useless. We can directly use the `len()` function.

It as been moved after the `self._option.append()` because originally the counter was incremented before adding a widget in `self._option`.